### PR TITLE
Doc dash as submodule

### PR DESCRIPTION
**Pull request by chrispsommers**
- Outlines a process for importing DASH project into another Git Project as a submodule. Feedback and suggestions are welcome!
- See sample project using this at https://github.com/chrispsommers/dashsubmodule

### CODE REVIEW

**Review by budgrise — APPROVED** [review]
looks good!

File: dash-pipeline/README-dash-as-submodule.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  - [Recap: DASH bmv2 workflow](#recap-dash-bmv2-workflow)
+  - [Custom DASH Workflow](#custom-dash-workflow)
+  - [Reusable Build toolchain and artifacts](#reusable-build-toolchain-and-artifacts)
+  - [Required Custom Tools and Artifacts](#required-custom-tools-and-artifacts)
+  - [Custom Traffic Test Harness](#custom-traffic-test-harness)
+  - [Custom Tests](#custom-tests)
+  - [Third-Party CI Pipeline Automation (Git Actions)](#third-party-ci-pipeline-automation-git-actions)
+# Importing the DASH project into another project
+
+The [Azure/DASH project](https://github.com/Azure/DASH) can be used as a resouce within other projects, such as third-party, commercial or open-source DASH implementations. For example, a commercial DPU vendor can incorporate the DASH project into a private Git repository and utilize many of the components, providing consistency with the community implementation and definition, reusing test-cases, and avoiding duplication of efforts.
```
Comment (vmytnykx) [inline dash-pipeline/README-dash-as-submodule.md:18]: resouce -> resource

File: dash-pipeline/README-dash-as-submodule.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+# Quick-Start
+This will show you how to import the [Azure/DASH project](https://github.com/Azure/DASH) project into your own Git project.
+
+A minimal sample project created using this recipe can be found here: https://github.com/chrispsommers/dashsubmodule
+
+
+1. Start with a Git project, either a new or existing one of your choosing. You might want to make a scratch project just to try this out.
+2. Copy the [Makefile.3rdpty](Makefile.3rdpty) into your project. You can put it into its own subdirectory and/or or rename it to suit. If you rename it, please interpret the subsequent instructions accordingly.
```
Comment (vmytnykx) [inline dash-pipeline/README-dash-as-submodule.md:27]: Need to fix ref to `Makefile.3rdpty` or move it into DASH

File: dash-pipeline/README-dash-as-submodule.md (hunk 1 above)
Comment (chrispsommers) [reply dash-pipeline/README-dash-as-submodule.md]: Thanks, fixed (along with many other typos and buzzwords once I merged from main and got Spellcheck running!).

File: dash-pipeline/README-dash-as-submodule.md (hunk 2 above)
Comment (chrispsommers) [reply dash-pipeline/README-dash-as-submodule.md:27]: Thanks, I'd forgotten to copy it into DASH repo.

**Review by vmytnykx — APPROVED** [review]

**Comment by KrisNey-MSFT** [comment]
per Community call

**Pull request merged**